app/code/Mykhailok/AskAboutThisProduct/view/frontend/web/js/view/question.js

```diff
@@ -0,0 +1,55 @@
+define([
+    'jquery',
+    'ko',
+    'uiComponent',
+    'mykhailokQuestionActionSave',
+    'mage/cookies'
+], function ($, ko, Component, saveMessage) {
+    'use strict';
+
+    return Component.extend({
+        qNameValue: ko.observable(),
+        qEmailValue: ko.observable(),
+        qMessageValue: ko.observable(),
+
+        defaults: {
+            isVisible: true,
+            action: '',
+            gProductId: 0,
+        },
+
+        /**
+         * @inheritdoc
+         */
+        initialize: function () {
+            this._super();
+        },
+
+        initObservable: function () {
+            return this._super()
+                .observe([
+                    'isVisible',
+                ])
```

Comment:
@mykhailok-dv `;` missed at the end of the instruction here and in the `toggleForm` method